src/lib/EventListener/AdminExceptionListener.php

```diff
@@ -49,6 +49,9 @@ class AdminExceptionListener implements LoggerAwareInterface
     /** @var string */
     protected $kernelEnvironment;
 
+    /** @var mixed */
```

Comment:
I know that PSR does not contain this information in their phpdoc entries, but we should limit these values:

```suggestion
    /** @var \Psr\Log\LogLevel::* */
```

Indicating that we expect these values to be one of the constants defined in that class (which is true).

Reply:
Fixed in https://github.com/ezsystems/ezplatform-admin-ui/pull/2028/commits/7b852c64c76ed70521190e238b475459f36767fc